src/appleseed-max-impl/appleseedrenderer/resource.h

```diff
@@ -2,80 +2,78 @@
-#define IDC_COMBO_FILTER                            207
-#define IDC_TEXT_FILTER_SIZE                        208
-#define IDC_SPINNER_FILTER_SIZE                     209
-#define IDS_RENDERERPARAMS_FILTER_TYPE_1            210
-#define IDS_RENDERERPARAMS_FILTER_TYPE_2            211
-#define IDS_RENDERERPARAMS_FILTER_TYPE_3            212
-#define IDS_RENDERERPARAMS_FILTER_TYPE_4            213
-#define IDS_RENDERERPARAMS_FILTER_TYPE_5            214
-#define IDS_RENDERERPARAMS_FILTER_TYPE_6            215
-#define IDS_RENDERERPARAMS_FILTER_TYPE_7            216
-#define IDS_RENDERERPARAMS_FILTER_TYPE_8            217
-
-#define IDD_FORMVIEW_RENDERERPARAMS_LIGHTING        300
-#define IDC_CHECK_GI                                301
-#define IDC_CHECK_CAUSTICS                          302
-#define IDC_STATIC_BOUNCES                          303
-#define IDC_TEXT_BOUNCES                            304
-#define IDC_SPINNER_BOUNCES                         305
-#define IDC_CHECK_MAX_RAY_INTENSITY                 306
-#define IDC_TEXT_MAX_RAY_INTENSITY                  307
-#define IDC_SPINNER_MAX_RAY_INTENSITY               308
-#define IDC_CHECK_BACKGROUND_EMITS_LIGHT            309
-#define IDC_STATIC_BACKGROUND_ALPHA                 310
-#define IDC_TEXT_BACKGROUND_ALPHA                   311
-#define IDC_SPINNER_BACKGROUND_ALPHA                312
-#define IDC_CHECK_FORCE_OFF_DEFAULT_LIGHT           313
-
-#define IDD_FORMVIEW_RENDERERPARAMS_OUTPUT          400
-#define IDC_RADIO_RENDER                            401
-#define IDC_RADIO_SAVEPROJECT                       402
-#define IDC_RADIO_SAVEPROJECT_AND_RENDER            403
-#define IDC_STATIC_PROJECT_FILEPATH                 404
-#define IDC_TEXT_PROJECT_FILEPATH                   405
-#define IDC_BUTTON_BROWSE                           406
-#define IDC_STATIC_SCALE_MULTIPLIER                 407
-#define IDC_TEXT_SCALE_MULTIPLIER                   408
-#define IDC_SPINNER_SCALE_MULTIPLIER                409
-
-#define IDD_FORMVIEW_RENDERERPARAMS_SYSTEM          500
-#define IDC_TEXT_RENDERINGTHREADS                   501
-#define IDC_SPINNER_RENDERINGTHREADS                502
-#define IDC_CHECK_LOW_PRIORITY_MODE                 503
-#define IDC_CHECK_USE_MAX_PROCEDURAL_MAPS           504
-#define IDC_CHECK_LOG_MATERIAL_EDITOR               505
-
-#define IDD_DIALOG_LOG                              600
-#define IDC_COMBO_LOG                               601
-#define IDC_STATIC_LOG                              602
-#define IDC_BUTTON_LOG                              603
-#define IDC_EDIT_LOG                                604
-#define IDC_CHECK_RENDER_STAMP                      605
-#define IDC_TEXT_RENDER_STAMP                       606
-#define IDS_RENDERERPARAMS_LOG_OPEN_MODE_1          607
-#define IDS_RENDERERPARAMS_LOG_OPEN_MODE_2          608
-#define IDS_RENDERERPARAMS_LOG_OPEN_MODE_3          609
+#define IDD_FORMVIEW_RENDERERPARAMS_ABOUT 100
```

Comment:
Formatting lost in this file...

Reply:
I changed the formatting as you asked (replaced all tabs with spaces, regrouped  etc.) in the latest 2 commits of this PR.